apisix/patch.lua

```diff
@@ -86,6 +93,64 @@ do
 end
 
 
+do -- `math.randomseed` patch
+
+    -- Seeds the random generator, use with care.
+    -- Once - properly - seeded, this method is replaced with a stub
+    -- one. This is to enforce best-practices for seeding in ngx_lua,
+    -- and prevents third-party modules from overriding our correct seed
+    -- (many modules make a wrong usage of `math.randomseed()` by calling
+    -- it multiple times or by not using unique seeds for Nginx workers).
+    -- Inspired by kong.globalpatches
```

Comment:
if these codes are copied from kong, you MUST add copyright in https://github.com/apache/apisix/blob/master/LICENSE.
if not, please remove this line.